parsers/mcmaster-parser-2.js

```diff
@@ -102,9 +102,16 @@ function parse(text){
     return timetable;
 }
 
-fs.readFile(filename, 'utf8', function(err, data) {
-    if (err) throw err;
-    console.log(ics.buildICS(parse(data)));
-});
+var convertToCal = function(rawTimetable, directory, filename, callback){
```

Comment:
this function should be the same for all parsers so why not just call:
```
var icsFileContent = ics.buildICS(parse(rawTimetable));
fs.writeFile(directory + filename, icsFileContent);
```
from wherever you call convertToCal?